ktor/src/nativeMain/kotlin/dev/evo/prometheus/ktor/native.kt

```diff
@@ -0,0 +1,6 @@
+package dev.evo.prometheus.ktor
+
+import dev.evo.prometheus.EmptyProcessMetrics
+import dev.evo.prometheus.ProcessMetrics
+
```

Comment:
Had no ideas how to provide them at this point in native so left empty just for the time being. The only way to get similar data in native I'm aware of is: https://github.com/ncabatoff/process-exporter